### PR TITLE
Fixed the rounded corners of the buttons in the RenameDialog & AddItemDialog

### LINKED ISSUE #2491: Border radius not applied
**Describe the bug**
Buttons (not selected/focussed) don't have the border-radius, they are "squared", as you can see in the screenshots

**To Reproduce**
1. Go to a generic Folder
2. Click on '[three dots at the top-right]'
3. Click on "New" then "Text document"
4. See the error

**Expected behavior**
The buttons should have the border radius

**Screenshots**
image
image

**Desktop (please complete the following information):**
 - OS Version: Windows 10 Pro 20H2 19042.630
 - App version: v0.21.0.0
**Additional context**
Add any other context about the problem here. Does this problem occur again after restarting the app?

### PR DESCRIPTION
**Pull request by Thomas1664**
Fixes #2491 using DefaultButtonStyle

### CODE REVIEW

**Comment by yaichenbaum** [comment]
@Thomas1664 It might be a good idea to use the accent color style for the primary buttons.

File: Files/Dialogs/AddItemDialog.xaml [hunk 1]
```diff
@@ -13,6 +13,7 @@
     BorderThickness="0"
     CornerRadius="4"
     PrimaryButtonText="Cancel"
+    PrimaryButtonStyle="{StaticResource DefaultButtonStyle}"
```
Comment (yaichenbaum) [inline Files/Dialogs/AddItemDialog.xaml:16]:
```suggestion
    PrimaryButtonStyle="{StaticResource AccentButtonStyle}"
```

Comment (Thomas1664) [reply Files/Dialogs/AddItemDialog.xaml:16]: The button in AddItemDialog didn't have primary color previously, so I didn't change it.

Comment (yaichenbaum) [reply Files/Dialogs/AddItemDialog.xaml:16]: That must have been an oversight.

**Review by yaichenbaum — APPROVED** [review]

**Pull request merged**